src/main/java/org/mockito/internal/util/reflection/InstrumentationMemberAccessor.java

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright (c) 2020 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.internal.util.reflection;
+
+import net.bytebuddy.ByteBuddy;
+import net.bytebuddy.agent.ByteBuddyAgent;
+import net.bytebuddy.dynamic.loading.ClassLoadingStrategy;
+import net.bytebuddy.implementation.MethodCall;
+import org.mockito.exceptions.base.MockitoInitializationException;
+import org.mockito.plugins.MemberAccessor;
+
+import java.lang.instrument.Instrumentation;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+import java.lang.reflect.*;
+import java.util.*;
+
+import static net.bytebuddy.matcher.ElementMatchers.named;
+import static org.mockito.internal.util.StringUtil.join;
+
+class InstrumentationMemberAccessor implements MemberAccessor {
```

Comment:
There's quite a bit code there and I think the code coverage is not high enough, which is what CodeCov is catching. Is it feasible to increase the coverage here or is that not possible?

Reply:
I can add a few tests for the exception scenarios.